mypy/types.py

```diff
@@ -171,7 +171,7 @@ def deserialize_type(data: Union[JsonDict, str]) -> 'Type':
     raise NotImplementedError('unexpected .class {}'.format(classname))
 
 
-class Type(mypy.nodes.Context):
+class Type(mypy.nodes.Context, ABC):
```

Comment:
This is not required, we can catch missing `@abstractmethod`s with `mypy` itself.

Reply:
Hmm, interesting this is one place mypy/pyright differs https://github.com/microsoft/pyright/issues/2547. Pyright requires ABC as during runtime the error is only raised for ABC.

I'm fine dropping ABC, just wanted to note if you want run other type checkers on mypy's codebase this will make a difference.